.github/workflows/update-configuration.yml

```diff
@@ -0,0 +1,58 @@
+name: "Update Configuration"
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  update:
+    name: "Update Configuration in manifest.json"
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: Install deps and run configuration update
+        run: |
+          yarn install --immutable --immutable-cache --check-cache
+          yarn tsc --noCheck --project tsconfig.json
+
+      - name: Update manifest configuration using GitHub Script
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+
+            const { pluginSettingsSchema } = require('./src/types/plugin-inputs');
+
+            const manifestPath = path.resolve("${{ github.workspace }}", './manifest.json');
+            const manifest = JSON.parse(fs.readFileSync(manifestPath, 'utf8'));
+
+            const configuration = JSON.stringify(pluginSettingsSchema);
+
+            manifest["configuration"] = JSON.parse(configuration);
+
+            const updatedManifest = JSON.stringify(manifest, null, 2)
+            console.log('Updated manifest:', updatedManifest);
+            fs.writeFileSync(manifestPath, updatedManifest);
+
+      - name: Commit and Push generated types
+        run: |
+          git config --global user.name 'ubiquity-os[bot]'
+          git config --global user.email 'ubiquity-os[bot]@users.noreply.github.com'
```

Comment:
I noticed some "unverified" tags on the commits from this, if it is not installed in the repository. Perhaps we should make a dedicated app, similar to our `ubiquity-os-continuous-deploys` that doesn't do anything except for authenticates properly. Then we can safely add it to every org and not have that unsightly `unverified` badge.

On GitHub iOS it displays even worse, in red, saying its a mismatched signature.

Reply:
Interesting enough, there is a long thread on that topic going on: https://github.com/actions/runner/issues/667